book/Makefile

```diff
@@ -11,7 +11,7 @@ strict:
 	cd website && jupyter-book build --html --strict
 
 ci:
-	cd website && jupyter-book build --html --ci
+	cd website && jupyter-book build --html --ci --strict
```

Comment:
Were you hoping to get it to pass with the `--strict` flag @sgibson91?
I've removed it for now since we seem to be down to some DOI parsing issues.
My proposal would be that we merge without it but with the agreement that we'll immediately set up a following PR to reintroduce it and monitor progress towards it passing the strict conditions.

Reply:
I think the plan was to do that in another PR. There are a number of warnings to fix and it would slow down the migration/make resolving conflicts very difficult.